src/components/Header.js

```diff
@@ -0,0 +1,40 @@
+import { Link } from "react-router-dom";
+import {useCart} from "../contexts/cart-context/cart-context"
+import {useWishlist} from "../contexts/wishlist-context/wishlist-context"
```

Comment:
You can create a index.js file in context directory as well to make use of barrel imports